src/clarity-angular/popover/signpost/signpost.ts

```diff
@@ -26,14 +29,19 @@ const signpostPositions: string[] = [
 @Component({
     selector: "clr-signpost",
     template: `
-        <button
-            #anchor
-            type="button"
-            class="signpost-action btn btn-small btn-link"
-            [ngClass]="{active: open}"
-            clrSignpostTrigger>
-            <clr-icon shape="info"></clr-icon>
-        </button><!-- TODO: turn into ng-content to handle default signpost icon and clrSignpostTrigger @Input. -->
+        <div class="signpost-trigger" #anchor>
+            <ng-content></ng-content>
+            <ng-container *ngIf="!useCustomTrigger">
+                <button
+                    type="button"
+                    class="signpost-action btn btn-small btn-link"
+                    [ngClass]="{active: open}"
+                    clrSignpostTrigger>
+                    <clr-icon shape="info"></clr-icon>
+                </button>
+            </ng-container>
+        </div>
+
         <ng-template [(clrPopover)]="ifOpenService.open"
```

Comment:
Oh, I just realized this is pre-Jeeyun's PR. You're going to have a few conflicts.